pycrostates/viz/cluster_centers.py

```diff
@@ -114,6 +142,24 @@ def plot_cluster_centers(
             ax = axes[i, j]
         # plot
         plot_topomap(center, info, axes=ax, show=False, **kwargs)
+        # Add min max vector
+        if show_gradient:
+            i_min = np.argmin(center)
+            i_max = np.argmax(center)
+            pos = _find_topomap_coords(info, picks="all")
+            if gradient_kwargs is not None:
+                ax.plot(
+                    [pos[i_min, 0], pos[i_max, 0]],
+                    [pos[i_min, 1], pos[i_max, 1]],
+                    "P-k",
```

Comment:
One more point, `"P-k"` is the `fmt` style of the line/marker, right? Maybe that should be passed as the default kwargs: `kwargs=dict(fmt="P-k")` or else we are risking a conflict if someone passes a different `fmt` in the kwargs.